envoy_iptables/proxy_init.sh

```diff
@@ -6,9 +6,11 @@ set -o nounset
 set -o pipefail
 
 usage() {
-  echo "${0} -p PORT -u UID [-h]"
+  echo "${0} -p INBOUND_PORT -o OUTBOUND_PORT -u UID [-h]"
   echo ''
-  echo '  -p: Specify the envoy port to which redirect all TCP traffic'
+  echo '  -p: Specify the envoy port to which redirect all inbound TCP traffic'
+  echo '  -o: Specify the envoy port to which redirect all outbound TCP traffic'
+  echo '      (optional: defaults to the value provided for the inbound port)'
```

Comment:
If the user does not want to redirect external traffic to Envoy that is something not possible, correct ?

Assume if the user sets only the `-p` option and now the outbound traffic is forwarded to Envoy, will Envoy send it out ? Afaik it would need the `ORIGINAL_DST` filter to be specified. Is that something you've tried out ?

Reply:
thanks for pointing that out. changed it to set outbound conditionally only if `-o` option is present